nextcord/ext/commands/bot.py

```diff
@@ -949,6 +963,137 @@ def reload_extension(self, name: str, *, package: Optional[str] = None) -> None:
             sys.modules.update(modules)
             raise
 
+    def load_extensions(
+        self,
+        names: List[str],
+        *,
+        package: Optional[str] = None,
+        packages: Optional[List[str]] = None,
+        extras: Optional[List[Dict[str, Any]]] = None,
+    ) -> List[str]:
+        """Loads all extensions provided in a list.
+
+        Parameters
+        ----------
+        names: List[:class:`str`]
+            The names of all of the extensions to load.
+        package: Optional[:class:`str`]
+            The package name to resolve relative imports with.
+            This is required when loading an extension using a relative path, e.g ``.foo.test``.
+            Defaults to ``None``.
+        packages: Optional[List[:class:`str`]]
+            A list of packages to resolve relative imports with.
+            This is required when loading an extension using a relative path, e.g ``.foo.test``.
+            Defaults to ``None``.
+        extras: Optional[List[Dict[:class:`str`, Any]]]
+            A list of extra arguments to pass to the extension's setup function.
+
+        Returns
+        --------
+        A List[:class:`str`] that contains the names of all of the extensions
+        that loaded successfully.
+
+        Raises
+        --------
+        ValueError
+            - If the ``package`` and ``packages`` parameters are both provided
+            - If the length of ``packages`` or the length of ``extras` is not equal to the length of ``names``
+        ExtensionNotFound
+            An extension could not be imported.
+        ExtensionAlreadyLoaded
+            An extension is already loaded.
+        NoEntryPointError
+            An extension does not have a setup function.
+        ExtensionFailed
+            An extension or its setup function had an execution error.
+        """
+        if package and packages:
+            raise ValueError("Cannot provide both package and packages.")
```

Comment:
imo raise an InvalidArgument is better

```suggestion
            raise errors.InvalidArgument("Cannot provide both package and packages.")
```